scripts/build.sh

```diff
@@ -41,8 +45,12 @@ constructor $TEMP_DIR/Miniforge3/ --output-dir $TEMP_DIR $EXTRA_CONSTRUCTOR_ARGS
 
 echo "***** Generate installer hash *****"
 cd $TEMP_DIR
-# This line ill break if there is more than one installer in the folder.
```

Comment:
Keep this comment?

Reply:
ping @SylvainCorlay

Reply:
Ah, missed it. Is it important? We only build one installer anyways.

Reply:
Let's keep the comment

Reply:
Done